platformio.ini

```diff
@@ -267,17 +267,10 @@ framework         = arduino
 extra_scripts     = ${common.extra_scripts}
 build_flags       = ${common.build_flags}
 lib_deps          = ${common.lib_deps}
-platform_packages = platformio/tool-dfuutil@^1.11.0
 monitor_speed     = 250000
-monitor_flags     =
-  --quiet
-  --echo
-  --eol
-    LF
-  --filter
-    colorize
-  --filter
-    time
+monitor_eol       = LF
+monitor_echo      = yes
+monitor_filters   = colorize, time, send_on_enter, log2file, esp32_exception_decoder
```

Comment:
`esp32_exception_decoder` is only available if ESP32 dev-platform is installed. This filter is not a part of PlatformIO Core. It makes sense to declare it only for ESP-32-based dev-platforms.

Reply:
Ah, so it is not otherwise ignored? I had hoped so to keep this to one entry. Also, we do have some of that debugging special sauce added to some of the other HALs, so it's possible the same parsing could apply to those.